tools/check_ai_images.py

```diff
@@ -11,7 +11,7 @@
 logger = logging.getLogger(__name__)
 logging.getLogger("__main__").setLevel(logging.INFO)
 
-DEFAULT_REGISTRY = "quay.io/ocpmetal"
+DEFAULT_REGISTRY = "quay.io/edge-infrastructure"
```

Comment:
Is this used anywhere now?

Reply:
yes, by QE
I'll work together with @nshidlin to make the transition. I think it doesn't make sense to block on it because it seems co-depending with this change